lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -137,7 +137,8 @@ pyparsing==3.0.9 ; python_version >= "3.7" and python_version < "3.12"
 pyreadline3==3.4.1 ; sys_platform == "win32" and python_version >= "3.8" and python_version < "3.12"
 pyreadline==2.1 ; sys_platform == "win32" and python_version < "3.8" and python_version >= "3.7"
 pyrsistent==0.19.3 ; python_version >= "3.7" and python_version < "3.12"
-pysam==0.20.0 ; python_version >= "3.7" and python_version < "3.12"
+pysam==0.20.0 ; python_version >= "3.7" and python_version < "3.11"
+pysam==0.21.0 ; python_version >= "3.11" and python_version < "3.12"
```

Comment:
Can we just use pysam==0.21.0 for all Python versions? That's what we have on 23.1 and dev.

Reply:
Sure, wasn't aware of this. Thanks for the hint.